src/store/initial-state.js

```diff
@@ -17,7 +17,6 @@ export const createInitialState = () => ({
   prettyName: settings.prettyName.default,
   ignoreLargeWarning: false,
   loading: {
-    graph: false,
```

Comment:
This cannot be deleted as this is still used within the `loading` selectors to trigger the loader during graph calculations